Libraries/Animated/src/animations/SpringAnimation.js

```diff
@@ -65,11 +71,14 @@ class SpringAnimation extends Animation {
   _lastPosition: number;
   _fromValue: number;
   _toValue: any;
-  _tension: number;
-  _friction: number;
+  _stiffness: ?number;
```

Comment:
Can we make `_stiffness`, `_damping`, `_mass` and `_initialVelocity` non-nullable now? Looks like they are initialized in every case.